support/commands/hexo.ts

```diff
@@ -1,20 +1,21 @@
 import { join } from 'path';
 import { exec, promisify } from 'grunt-dojo2-extras/src/util/process';
 
-function generateHexo(siteDirectory: string, configs: string[] = [ '_config.yml' ]) {
+function generateHexo(siteDirectory: string, configs: string[] = [ '_config.yml' ], flags: string[]) {
```

Comment:
We should either remove the default value on `configs` or add a default to `flags`; i.e. `flags: string[] = []`. It doesn't matter which one we do since `generateHexo` is internal, but we should maintain consistency.